### PR TITLE
add dockerfiles for tf 1.15.2 py37 containers

### PR DESCRIPTION
**Pull request by Satish615**
*Description of changes:*
This pr is to support dlc containers for tensorflow 1.15.2 with python 3.7

1. installed python 3.7 from source on ubuntu 18.04 container.
2. installed tensorflow 2.2 from pypi - to run the ec2, ecs and eks tests (temporary change)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docker/1.15.2/py37/Dockerfile.cpu [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+# See http://bugs.python.org/issue19846
+ENV PYTHONIOENCODING=UTF-8
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+# Specify the location of module that contains the training logic for SageMaker
+# https://docs.aws.amazon.com/sagemaker/latest/dg/docker-container-environmental-variables-entrypoint.html
+ENV SAGEMAKER_TRAINING_MODULE=sagemaker_tensorflow_container.training:main
+
+# Define framework-related package sources
+# To be changed for py3.7
```
Comment (saimidu) [inline docker/1.15.2/py37/Dockerfile.cpu:25]: Can remove this line

File: docker/1.15.2/py37/Dockerfile.cpu [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+ && cp /root/.ssh/id_rsa.pub /root/.ssh/authorized_keys \
+ && printf "Host *\n  StrictHostKeyChecking no\n" >> /root/.ssh/config
+
+WORKDIR /
+
+
+ARG PYTHON=python3
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.7.7
```
Comment (saimidu) [inline docker/1.15.2/py37/Dockerfile.cpu:82]: Please move this to the top to be right after all the ENV definitions.

File: docker/1.15.2/py37/Dockerfile.cpu [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends libreadline-gplv2-dev \
+    libncursesw5-dev libffi-dev libssl-dev libsqlite3-dev tk-dev \
+    libgdbm-dev libc6-dev libbz2-dev \
+    && apt-get clean
+
+
```
Comment (saimidu) [inline docker/1.15.2/py37/Dockerfile.cpu:90]: Is there a reason to set two-line spaces between statements on the dockerfile?

File: docker/1.15.2/py37/Dockerfile.cpu [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+ARG PYTHON=python3
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.7.7
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends libreadline-gplv2-dev \
+    libncursesw5-dev libffi-dev libssl-dev libsqlite3-dev tk-dev \
+    libgdbm-dev libc6-dev libbz2-dev \
+    && apt-get clean
```
Comment (saimidu) [inline docker/1.15.2/py37/Dockerfile.cpu:88]: Please reorder and format `apt-get install` statements to follow Dockerfile Best Practices.
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments

Please place each package in its own line, and sort them in lexicographical order.

Indent mutli-line statements in the following manner:
```
RUN abcd \
 && efgh \
    ijkl \
    mnop \
 && qrst
```

Also, please add `rm -rf /var/lib/apt/lists/*` after `apt-get clean`, similar to the first `apt-get install` statement.

File: docker/1.15.2/py37/Dockerfile.cpu [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+
+RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz && \
+    tar -xvf Python-$PYTHON_VERSION.tgz && cd Python-$PYTHON_VERSION && \
+    ./configure && make && make install && \
+    make && make install && rm -rf ../Python-$PYTHON_VERSION*
```
Comment (saimidu) [inline docker/1.15.2/py37/Dockerfile.cpu:94]: Break into multiple lines following formatting rules as seen in other dockerfiles in this repository.

File: docker/1.15.2/py37/Dockerfile.gpu [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends --allow-unauthenticated \
+    libreadline-gplv2-dev libncursesw5-dev libffi-dev libssl-dev \
+    libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev \
+    && apt-get clean
+
+RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz && \
+    tar -xvf Python-$PYTHON_VERSION.tgz && cd Python-$PYTHON_VERSION && \
+    ./configure && make && make install && \
+    make && make install && rm -rf ../Python-$PYTHON_VERSION*
```
Comment (saimidu) [inline docker/1.15.2/py37/Dockerfile.gpu:129]: Same comments as CPU dockerfile

File: docker/1.15.2/py37/Dockerfile.cpu (hunk 1 above)
Comment (saimidu) [reply docker/1.15.2/py37/Dockerfile.cpu:25]: this line == `# To be changed for py3.7`

File: docker/1.15.2/py37/Dockerfile.gpu (hunk 6 above)
Comment (Satish615) [reply docker/1.15.2/py37/Dockerfile.gpu]: made the changes

File: docker/1.15.2/py37/Dockerfile.cpu (hunk 4 above)
Comment (Satish615) [reply docker/1.15.2/py37/Dockerfile.cpu]: moved them to seperate lines in lexicographical order.

File: docker/1.15.2/py37/Dockerfile.cpu (hunk 3 above)
Comment (Satish615) [reply docker/1.15.2/py37/Dockerfile.cpu]: remove extra lines

File: docker/1.15.2/py37/Dockerfile.cpu (hunk 2 above)
Comment (Satish615) [reply docker/1.15.2/py37/Dockerfile.cpu]: moved to the top below TF_URL ARGS

File: docker/1.15.2/py37/Dockerfile.cpu (hunk 1 above)
Comment (Satish615) [reply docker/1.15.2/py37/Dockerfile.cpu]: removed the comment

**Pull request merged**